recipe_scrapers/__init__.py

```diff
@@ -174,6 +174,7 @@
 from .fortyaprons import FortyAprons
 from .franzoesischkochen import FranzoesischKochen
 from .g750g import G750g
+from .garnishandglaze import garnishandglaze
```

Comment:
```suggestion
from .garnishandglaze import GarnishAndGlaze
```

Class names should follow CamelCase formatting like in your scraper class.